pkg/instance/tshark.go

```diff
@@ -46,6 +46,10 @@ func (i *Instance) createTsharkCollectorInstance(ctx context.Context) (*Instance
 		return nil, err
 	}
 
+	if i.tsharkCollectorConfig == nil {
+		return nil, ErrTsharkCollectorConfigNotSet
+	}
+
```

Comment:
This needs to be before `AddVolume` as well since it uses `i.tsharkCollectorConfig.VolumeSize`. 
imo this should be the first check of the method to prevent unnecessary actions prior to such a baseline requirement.